php/mixphp-swoole/composer.json

```
@@ -0,0 +1,10 @@
+{
+  "autoload": {
+    "psr-4": {
+      "App\\": "src/"
+    }
+  },
+  "require": {
+    "mix/vega": "~3.0"
```

```suggestion
    "mix/vega": "~3.0.0"
```

`~3.0` could lead to **3.1** 
https://getcomposer.org/doc/articles/versions.md#tilde-version-range-